src/util-time.h

```diff
@@ -24,13 +24,41 @@
 #ifndef __UTIL_TIME_H__
 #define __UTIL_TIME_H__
 
+typedef uint64_t SCTime_t;
```

Comment:
if we'd make this something like
```c
typedef struct {
  uint64_t secs:44;
  uint64_t usecs:20;
} SCTime_t;
```
We'd get masking for free I think (not at runtime, but code / review style overhead)

Reply:
Will do in a separate commit.